sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample2.1_AnalyzeSentimentWithOpinionMiningAsync.cs

```diff
@@ -33,33 +34,33 @@ public async Task AnalyzeSentimentWithOpinionMiningAsync()
                 "We changed rooms as the toilet smelled."
             };
 
-            AnalyzeSentimentResultCollection reviews = await client.AnalyzeSentimentBatchAsync(documents, options: new AnalyzeSentimentOptions() { AdditionalSentimentAnalyses = AdditionalSentimentAnalyses.OpinionMining });
+            AnalyzeSentimentResultCollection analyzedReviews = await client.AnalyzeSentimentBatchAsync(reviews, options: new AnalyzeSentimentOptions() { MineOpinions = true });
 
-            Dictionary<string, int> complaints = GetComplaint(reviews);
+            Dictionary<string, int> complaintTargets = GetNegativeOpinionTargets(analyzedReviews);
 
-            var negativeAspect = complaints.Aggregate((l, r) => l.Value > r.Value ? l : r).Key;
-            Console.WriteLine($"Alert! major complaint is *{negativeAspect}*");
+            var negativeOpinionTarget = complaintTargets.Aggregate((a, b) => a.Value > b.Value ? a : b).Key;
+            Console.WriteLine($"Alert! major complaint target is *{negativeOpinionTarget}*");
             Console.WriteLine();
-            Console.WriteLine("---All complaints:");
-            foreach (KeyValuePair<string, int> complaint in complaints)
+            Console.WriteLine("---All complaint targets:");
+            foreach (KeyValuePair<string, int> complaintTarget in complaintTargets)
             {
-                Console.WriteLine($"   {complaint.Key}, {complaint.Value}");
+                Console.WriteLine($"   OpinionTarget: {complaintTarget.Key}, Number of negative descriptions: {complaintTarget.Value}");
             }
         }
 
-        private Dictionary<string, int> GetComplaint(AnalyzeSentimentResultCollection reviews)
+        private static Dictionary<string, int> GetNegativeOpinionTargets(AnalyzeSentimentResultCollection reviews)
         {
             var complaints = new Dictionary<string, int>();
             foreach (AnalyzeSentimentResult review in reviews)
             {
                 foreach (SentenceSentiment sentence in review.DocumentSentiment.Sentences)
                 {
-                    foreach (MinedOpinion minedOpinion in sentence.MinedOpinions)
+                    foreach (MinedOpinion opinion in sentence.Opinions)
```

Comment:
Let's keep the type name MinedOpinion, but name the property Opinions, for clarity.  When the caller is reasoning about it, they don't need to think about whether it was mined or not.